_data/assetPaths.json

```diff
@@ -1,11 +1,26 @@
 {
```

Comment:
I'm curious -- do we gain anything by having assetPaths checked in? in another project, we're looking at adding it to the gitignore because its so noisy in diffs. If you agree, could you delete it and add it to the ignore?

Reply:
I have also been curious about this...  Let me get it ignored after the others have been merged.  I'll make a note.  Do we even need _site at all?

Reply:
Per Slack discussion, removing "_site" from tracking and adding an issue here for the Pages template: https://github.com/cloud-gov/pages-uswds-11ty/issues/78